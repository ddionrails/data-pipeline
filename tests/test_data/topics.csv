study,name,label,label_de,description,description_de,parent
some-study,some-topic,some-topic,some-topic,,,,
some-study,some-subtopic,some-subtopic,some-subtopic,,,some-topic