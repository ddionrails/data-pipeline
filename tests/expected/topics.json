[
  {
    "language": "en",
    "topics": [
      {
        "title": "some-topic",
        "key": "topic_some-topic",
        "type": "topic",
        "children": [
          {
            "title": "some-concept",
            "key": "concept_some-concept",
            "type": "concept"
          }
        ]
      },
      {
        "title": "some-subtopic",
        "key": "topic_some-subtopic",
        "type": "topic",
        "children": []
      }
    ]
  },
  {
    "language": "de",
    "topics": [
      {
        "title": "",
        "key": "topic_some-topic",
        "type": "topic",
        "children": [
          {
            "title": "some-concept",
            "key": "concept_some-concept",
            "type": "concept"
          }
        ]
      },
      {
        "title": "",
        "key": "topic_some-subtopic",
        "type": "topic",
        "children": []
      }
    ]
  }
]